### ISSUE TITLE
Activity bar focus indication is too subtle

### ISSUE THREAD

**Issue by joaomoreno** [Microsoft/vscode#2376]
I find it very hard to notice when the focus is in the activity bar. We need something stronger there. Maybe try white? Or maybe some other affordance at all, since this is too far away from the user's eye.

![image](https://cloud.githubusercontent.com/assets/22350/12585511/a901ed88-c44c-11e5-8562-51ba91341176.png)

Additionally, notice how the opacity of the *unselected* viewlet icons affects the focus indication. The explorer's focus indication is brighter than the search one. They should be exactly the same.

![bug](https://cloud.githubusercontent.com/assets/22350/12585360/06641376-c44c-11e5-9027-6c7dc7cd9f3c.gif)

Related to #2145

ping @bgashler1

**Comment by dbaeumer** [comment]
Verified that icon turns white and that the blue bar is better visible.
